libcodechecker/cmd/cmd_line_client.py

```diff
@@ -105,7 +106,10 @@ def add_filter_conditions(report_filter, filter_str):
 
 
 def handle_list_runs(args):
+
     logger.setup_logger(args.verbose)
+    global LOG
```

Comment:
Couldn't we create a separate function for these steps at the top of the file?
E.g.:
```py
def init_logger(log_level):
  logger.setup_logger(log_level)
  global LOG
  LOG = logger.get_logger('system')
```

Reply:
done